src/components/PreviewList.js

```diff
@@ -91,6 +91,8 @@ const PreviewList = ({
             className={clsx(previewGridClasses.gridListTile, {
               [classes.iconWrapper]: !isImage,
             })}
+            onClick={handlePreviewClick(i)}
+            onKeyDown={handlePreviewClick(i)}
```

Comment:
Why did you set a listener on `KeyDown`?

Reply:
Hello, sorry for my late answer. I added the onKeyDown to allow user navigate with keyboard to increase the accessibility. Is it wrong?

Reply:
> Hi @cadrimiranda ,
> 
> Thanks for your contribution, LGTM 👍 .
> 
> Can you please remove the files under `dist` from the PR?

Sure, I'll remove.

Reply:
Done!